reaction_roles.js

```diff
@@ -0,0 +1,42 @@
+const Discord = require("discord.js")
+const mongoose = require('mongoose');
+const conf = require('./conf.js');
+mongoose.connect(conf.mongodbs, { useNewUrlParser: true })
+const reaction_message_model = require("./models/reaction_roles_message.js")
+
+/**
+ * @param {Discord.Client} bot discord client
+ * @param {event} event raw event;
+ * bot.on("raw", event => {}
+ */
+module.exports.run = async (bot, event) => {
+    var guild_obj = bot.guilds.find(r => r.id == event.d.guild_id)
+    var user_guild_member = await guild_obj.fetchMember(event.d.user_id).catch(err => console.log(err))
+    if(user_guild_member.user.bot) return
+    var message_id = event.d.message_id
+    // fetch data from database
+    reaction_message_model.findOne({ messageID: message_id },
+        async (err, rec) => {
```

Comment:
Unless necessary (i.e. to avoid rebinding of this) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.
```suggestion
        async function (err, rec) {
```